exporter/collector/googlemanagedprometheus/extra_metrics.go

```diff
@@ -17,26 +17,13 @@ package googlemanagedprometheus
 import (
 	"time"
 
-	"go.opentelemetry.io/collector/featuregate"
+	"github.com/prometheus/common/model"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	semconv "go.opentelemetry.io/collector/semconv/v1.18.0"
 )
 
-const (
-	// Special attribute key used by Ops Agent prometheus receiver to denote untyped
-	// prometheus metric. Internal use only.
-	GCPOpsAgentUntypedMetricKey = "prometheus.googleapis.com/internal/untyped_metric"
```

Comment:
will this break consistency/compatibility with the ops agent? Are they supporting this semconv yet?

Reply:
As long as they update their prometheus receiver when they update this exporter, they won't break.  Thats partially why I want Ridwan to review.

Reply:
+1 to what David said, this shouldn't break the Ops Agent, we plan on unforking all prometheus components at once